integration-tests/source-s3/gatsby-config.js

```diff
@@ -9,12 +9,14 @@ module.exports = {
       resolve: `gatsby-source-s3`,
       options: {
         aws: {
-          accessKeyId: "minioadmin",
-          secretAccessKey: "minioadmin",
-          s3ForcePathStyle: true,
-          endpoint: "localhost:9001",
+          credentials: {
+            accessKeyId: "minioadmin",
+            secretAccessKey: "minioadmin",
+          },
+          forcePathStyle: true,
+          endpoint: "http://localhost:9001",
           sslEnabled: false,
-          s3ForcePathStyle: true,
+          region: "us-east-1",
```

Comment:
Why was this added?

Reply:
From what I understand, v3 doesn't support a default region for custom endpoints. There's this comment which references a feature request to add the functionality from a few years ago https://github.com/aws/aws-sdk-js-v3/issues/1845#issuecomment-754832210, but that feature hasn't been added yet.

The region property comes from the `RegionInputConfig` interface which denotes the property as optional with no default. Since the integration test is using a custom endpoint (localhost), the region property has to be in the config otherwise an error is thrown.